app/javascript/actionsPanel/actionsPanel.js

```diff
@@ -345,18 +345,16 @@ export function addAdjustTagListeners() {
     },
   );
 
-  const adjustTagSubmitBtn = document.getElementById('tag-adjust-submit');
-  if (adjustTagSubmitBtn) {
-    adjustTagSubmitBtn.addEventListener('click', (e) => {
+  const form = document.getElementById('tag-adjust-submit')?.form;
+  if (form) {
+    form.addEventListener('submit', (e) => {
       e.preventDefault();
-      const textArea = document.getElementById('tag-adjustment-reason');
+
       const dataSource =
-        document.querySelector('button.adjustable-tag.active') ||
+        document.querySelector('button.adjustable-tag.active') ??
```

Comment:
TIL about `??` nullish operator!

Reply:
> TIL about ?? nullish operator!

Same here, @Zhao-Andy! 🤩

Reply:
I made the change here as it made more sense than an or. In this scenario both would work, but since `??` is supported now...